sys/include/net/sock/dtls.h

```diff
@@ -694,6 +711,46 @@ ssize_t sock_dtls_send(sock_dtls_t *sock, sock_dtls_session_t *remote,
  */
 void sock_dtls_close(sock_dtls_t *sock);
 
+/**
+ * @brief Creates a new DTLS session
+ *
+ * Sends a ClientHello to initiate a handshake process with a DTLS server  at
```

Comment:
```suggestion
 * Sends a ClientHello message to initiate a handshake process with a DTLS server  at
```
?

Reply:
Reworded the description to also mention this functions waits until handshake complete or timed out.